flipperdevices/Flipper-Android-App
# Infrared design and feature improvements

**Background**

This PR introduces various fixes and improvements into remote controls feature

**Changes**

- Fix button colors
- Fix button placeholders
- Add new illustration into setup
- Add notification after remote found/added
- Add search into brands
- Add dialog to skip/go back when sync is in progress

**Test plan**

- Open remote controls feature 
- Try find your remote
- See new fixed ui and other  listed improvements
